manifest_staging/charts/secretsync/README.md

```diff
@@ -0,0 +1,38 @@
+# Deploying the SecretSync Controller
```

Comment:
In a follow-up let's change all the references to "Secrets Store Sync Controller"

Reply:
https://github.com/kubernetes-sigs/secrets-store-sync-controller/issues/20